xarray/tests/test_cftimeindex_resample.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import absolute_import
+
+import pytest
+
+import numpy as np
+import pandas as pd
+import xarray as xr
+import pandas.core.resample as pdr
+import xarray.core.resample_cftime as xrr
+from pandas.tseries.frequencies import to_offset as pd_to_offset
+from xarray.coding.cftime_offsets import (to_cftime_datetime,
+                                          to_offset as xr_to_offset)
+
```

Comment:
Please add a couple module-level skips at the top here for these tests (which depend on both cftime and pandas >= 0.24):
```python
pytest.importorskip('cftime')
pytest.importorskip('pandas', minversion='0.24')
```
This will prevent the tests from being run on platforms where they will be expected to fail (due to either import errors or version-dependent behavior).

Reply:
Changing this to 0.24 keeps me from running the test since my pandas-dev version of pandas is  0.24.0.dev0+1555.gea8c9bf51.dirty, would 0.23.99 be an acceptable workaround?

Reply:
Sorry, yes, for now that would be ok.

Reply:
pandas 0.24 is out, so let's go ahead and update the `minversion` here.  The tests should now run on our main CI builds too.  Maybe wait until #2720 is merged before pushing, so you don't hit the unrelated failures.